src/alloc_pool.c

```diff
@@ -280,11 +282,12 @@ static void qd_alloc_init(qd_alloc_type_desc_t *desc)
         if (desc->additional_size)
             desc->total_size += *desc->additional_size;
 
-        if (desc->config == 0)
-            desc->config = desc->total_size > BIG_THRESHOLD ?
-                &qd_alloc_default_config_big : &qd_alloc_default_config_small;
-
-        assert (desc->config->local_free_list_max >= desc->config->transfer_batch_size);
+        desc->config = &qd_alloc_default_config_asan;
+//        if (desc->config == 0)
+//            desc->config = desc->total_size > BIG_THRESHOLD ?
+//                &qd_alloc_default_config_big : &qd_alloc_default_config_small;
+//
+//        assert (desc->config->local_free_list_max >= desc->config->transfer_batch_size);
```

Comment:
I think the assert is wrong; `local_free_list_max == 0` and `transfer_batch_size == 1` works just fine.